### DISCUSSION TITLE
Printing of historical alarms on demand

### DISCUSSION THREAD

**Discussion by robsori** [frangoteam/FUXA#1152] [Ideas]
Hi Fuxa team,
I believe that it would be necessary to print the historical alarm messages on a predetermined time interval at the request of the operator. There are situations in which the SCADA operator has the task of printing only the historical messages from a particular time interval. This function should be implemented on the real-time alarm control and not in the configuration mode where the operator must not have access.

**Comment by unocelli** [comment]
Hi,  creating a differentiated editor access is very expensive, it would take a configurator.... I think it could be solved via script by extending the system functions.
